### PR TITLE
ingest/build-configs/nextstrain-automation: Add README

### PR DESCRIPTION
**Pull request by joverlee521**
Add a README.md to describe the use of this build-config and to provide some instructions on how to use it for internal Nextstrain team members.

### CODE REVIEW

**Comment by joverlee521** [comment]
@j23414 and @kimandrews, please let me know if there are any additional instructions that would have been helpful for https://github.com/nextstrain/zika/pull/50 and https://github.com/nextstrain/measles/pull/19!

**Review by jameshadfield — APPROVED** [review]

File: ingest/build-configs/nextstrain-automation/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Provide the additional config file to the Snakemake options in order to
+include the custom rules from [upload.smk](upload.smk) in the workflow.
+Specify the `upload_all` target in order to run the additional upload rules.
+
+The upload rules will require AWS credentials for a user that has permissions
+to upload to the Nextstrain data bucket.
+
+The customized workflow can be run from the top level pathogen repo directory with:
+```
+nextstrain build
```
Comment (j23414) [inline ingest/build-configs/nextstrain-automation/README.md:27]: Minor edit, needs the "\\" in 
```
nextstrain build \
```

Comment (joverlee521) [reply ingest/build-configs/nextstrain-automation/README.md]: Ah, good eye! Fixed in [force-push](https://github.com/nextstrain/pathogen-repo-guide/compare/6a3ace1fbd829e5d8816ba9403273f749f6f60e7..dc9af57743150b2e434f51059e57c88b76d66d7a).

**Review by j23414 — APPROVED** [review]
Reads well to me :D

**Pull request merged**